tests/FindAndReplaceJsonTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+use Dcblogdev\FindAndReplaceJson\FindAndReplaceJson;
+use PHPUnit\Framework\TestCase;
+
+class FindAndReplaceJsonTest extends TestCase
+{
+
+    public function testReplaceWithJsonPayload()
+    {
+        $payload = json_encode(
+            [
+                'name' => 'Joe Bloggs',
+                'age' => 23,
+                'location' => 'London',
+                'foo' => [
+                    'bar' => 'baz',
+                ]
+            ]
+        );
+
+        $replaces = [
+            'age' => 45,
+            'location' => "Manchester",
+        ];
+
+        $expected = json_encode(
+            [
+                'name' => 'Joe Bloggs',
+                'age' => 45,
+                'location' => "Manchester",
```

Comment:
for this to pass it needs the other elements adding

```
$expected = json_encode(
    [
        'name' => 'Joe Bloggs',
        'age' => 45,
        'location' => "Manchester",
        'foo' => [
            'bar' => 'baz',
        ]
    ]
);
```

Reply:
Sure. I've removed the unnecessary elements